_data/navbar.yml

```diff
@@ -7,7 +7,7 @@
 - title: Docs & Resources
   href: /docs
 
-- title: 2021
+- title: 2021 
```

Comment:
Little whitespace got left here by mistake.
Not critical. Will be fixed in another commit.